docs/configuration.asciidoc

```diff
@@ -539,7 +534,7 @@ changing this setting to `false`. This setting is ignored when
 |============
 
 `ELASTIC_APM_LOG_FILE` specifies the output file for the agent's default, internal
-logger. The file will be created, or truncated if it exists, when the process starts.
+logger. The file will be created, or truncated if it exists when the process starts.
```

Comment:
This edit changes the meaning. With this change, the modifier "when the process starts" no longer relates grammatically to  "the file will be created." Was that intentional? This might be a good time to use dashes: The file will be created--or truncated, if it exists--when the process starts

Reply:
```suggestion
logger. The file will be created, or truncated if it exists, when the process starts.
```